public/javascripts/dashtabs.js

```diff
@@ -39,5 +39,8 @@
   window.tabs = tabs;
 })();
 
-const dashTabs = tabs();
-dashTabs.init();
\ No newline at end of file
+const dashTabs = tabs('tabs', 'tablink', 'tabcont');
+dashTabs.init();
+
```

Comment:
refactoring so the tabs function is reuseable, we have more than one tabs element on the page